pdf_on_submit/pdf_on_submit/doctype/enabled_doctype/enabled_doctype.json

```diff
@@ -65,17 +66,25 @@
    "fieldname": "filter_description",
    "fieldtype": "HTML",
    "options": "Optional: create PDF only if the submitted document matches these filters.<br><br>"
+  },
+  {
+   "description": "If set, the PDF will be stored in this specific Attach field. If not set, it will be attached to the document without a specific field.<br>\n<b>Note:</b> The Attach fields should be 'No Copy' and 'Read Only'. Otherwise they get overwritten on submit.",
+   "fieldname": "attach_to_field",
+   "fieldtype": "Data",
```

Comment:
This can be a "Select" or "Autocomplete" field with empty options. Then, in the frontend, we dynamically fetch the respective DocType's fields and set them as options.